app/assets/stylesheets/shared/button.scss

```diff
@@ -16,6 +16,19 @@
   }
 }
 
+.btn-secondary-link {
```

Comment:
the shared folder seems to be getting out of sync with Augury?

Reply:
Yeah, that's probably true.  Though I'm a bit surprised to see something in Augury and not in Feeder?  (I thought generally, Feeder CSS was way ahead of Augury now - and the plan was to backport Feeder's code to Augury...)